contracts/interfaces/external/balancer-v2/IWeightedPoolFactory.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+interface IWeightedPoolFactory {
```

Comment:
This interface was added here because we use it in our tests. It is not used in the integration. Is this the correct way to add it? Or should we add it somewhere else?

Reply:
yes, this is the best place. Thank you